prime-router/src/main/kotlin/TransportType.kt

```diff
@@ -126,6 +126,8 @@ data class RESTTransportType
     val tlsKeystore: String? = null,
     /** [parameters] The map of parameters to be sent in the message */
     val parameters: Map<String, String> = emptyMap(),
+    /** The map of parameters for constructing the jwt */
+    val jwtParams: Map<String, String> = emptyMap(),
```

Comment:
Why we can't use "parameters:" instead of introduce a new "jwtParams:"?

Reply:
Good question. I think "jwtParams:" is used specifically to customize the jwt. The "parameters:" map is used to construct the header. My thought process was to keep the responsibilities separated.